sources/platform/actors/running/tasks.md

```diff
@@ -1,51 +1,45 @@
-
-All your tasks are listed at [https://console.apify.com/actors/tasks](https://console.apify.com/actors/tasks).
+Actor tasks let you create multiple reusable configurations of a single Actor, adapted for specific use cases. For example, you can create one **Web Scraper** ([apify/web-scraper](https://apify.com/apify/web-scraper)) configuration (task) that scrapes the latest reviews from imdb.com, another that scrapes nike.com for the latest sneakers, and a third that scrapes your competitor's e-shop. You can then use and reuse these configurations directly from [Apify Console](https://console.apify.com/actors/tasks), [Schedules](../../schedules.md), or [API](/api/v2#/reference/actor-tasks/run-collection/run-task).
 
-You can create tasks both for your personal Actors and for those made by someone else. Just [search Apify Store](https://console.apify.com/store) to find an Actor you want to use and click the **Create new task** button.
-
-Like any other resource, you can share your Actor tasks with other Apify users via the [access rights](../../collaboration/index.md) system.
+You can find all your tasks at [https://console.apify.com/actors/tasks](https://console.apify.com/actors/tasks).
 
 ## Create
 
-Open any Actor from [Apify Store](https://console.apify.com/store) or your list of [Actors](https://console.apify.com/actors) in Apify Console. At the top-right section of the page, click the **Create empty task** button.
+To create a task, open any Actor from [Apify Store](https://console.apify.com/store) or your list of [Actors](https://console.apify.com/actors) in Apify Console. At the top-right section of the page, click the **Create task** button.
 
 ![Create a new Apify task](./images/tasks/create-task.png)
 
 ## Configure
 
-You can set up your task's input under the **Input and options** tab. A task's input configuration works just like an Actor's. After all, it is just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.
+You can set up your task's input under the **Input and options** tab. A task's input configuration works just like an Actor's. After all, it's just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.
 
 ![Apify task configuration](./images/tasks/create-task-configure.png)
 
-Actors' input fields will vary depending on their purpose, but they all follow the same principle: you provide an Actor with the information it needs so it can do what you want it to do.
+An Actors' input fields may vary depending on their purpose, but they all follow the same principle: _you provide an Actor with the information it needs so it can do what you want it to do._
 
 You can set run options such as timeout and [memory](./usage_and_resources.md) at the bottom of the input section.
 
-### Identifying tasks
+### Identification
```

Comment:
I personally prefer the old title here - sure it's longer, but I think it's clearer

Reply:
Here I tried to make sure that all the headings are in the similar style while still conveying the same information as. before

Reply:
I see what you mean, but the old version just reads easier. Really, though, shouldn't this section just be called `Naming`?

Reply:
Yup that works, my intention here was to keep the parallel structure of headings, but `Naming` definitely achieves the same result